Dockerfile-alpine.template

```diff
@@ -14,32 +16,35 @@ RUN ARCH= && alpineArch="$(arch)" \
     libstdc++ \
```

Comment:
Is this needed unless building?

Reply:
I think it is.

Reply:
I know you'd need it for building binaries (but that also requires other stuff), but maybe it's needed for execution?